libs/etterlatte-behandling-model/src/main/kotlin/no/nav/etterlatte/libs/common/Enheter.kt

```diff
@@ -2,21 +2,34 @@ package no.nav.etterlatte.common
 
 import no.nav.etterlatte.libs.common.person.Folkeregisteridentifikator
 
-enum class Enheter(val enhetNr: String, val navn: String, val inkludertForNasjonalTilgang: Boolean) {
-    STRENGT_FORTROLIG("2103", "NAV Vikafossen", false),
-    STRENGT_FORTROLIG_UTLAND("2103", "NAV Vikafossen", false),
-    EGNE_ANSATTE("4883", "NAV Familie- og pensjonsytelser Egne ansatte", false),
-    UTLAND("0001", "NAV Familie- og pensjonsytelser Utland", false),
-    AALESUND("4815", "NAV Familie- og pensjonsytelser Ålesund", true),
-    AALESUND_UTLAND("4862", "NAV Familie- og pensjonsytelser Ålesund Utland", false),
-    STEINKJER("4817", "NAV Familie- og pensjonsytelser Steinkjer", true),
-    PORSGRUNN("4808", "NAV Familie- og pensjonsytelser Porsgrunn", true),
+enum class Enheter(
+    val enhetNr: String,
+    val navn: String,
+    val inkludertForNasjonalTilgang: Boolean,
+    val lesetilgang: Boolean,
+    val skrivetilgang: Boolean,
+) {
+    STRENGT_FORTROLIG("2103", "NAV Vikafossen", false, true, true),
+    STRENGT_FORTROLIG_UTLAND("2103", "NAV Vikafossen", false, true, true),
+    EGNE_ANSATTE("4883", "NAV Familie- og pensjonsytelser Egne ansatte", false, true, true),
+    UTLAND("0001", "NAV Familie- og pensjonsytelser Utland", false, true, true),
+    AALESUND("4815", "NAV Familie- og pensjonsytelser Ålesund", true, true, true),
+    AALESUND_UTLAND("4862", "NAV Familie- og pensjonsytelser Ålesund Utland", false, true, true),
+    STEINKJER("4817", "NAV Familie- og pensjonsytelser Steinkjer", true, true, true),
+    PORSGRUNN("4808", "NAV Familie- og pensjonsytelser Porsgrunn", true, true, true),
+    OEST_VIKEN("4101", "NAV Kontaktsenter Øst-Viken", false, true, false),
+    TROENDELAG("4116", "NAV Kontaktsenter Trøndelag", false, true, false),
+    NORDLAND_BODOE("4118", "NAV Kontaktsenter Nordland Bodø", false, true, false),
```

Comment:
Hvis vi legger opp til en flyt der rettigheter baserer seg på enheten de jobber på blir vi veldig avhengig av at slike enum-klasser er oppdatert og holdes i synk 🤔 

Er dette en uttømmende liste av kontaktsenterene som skal ha lesetilgang? Google gir men en fin liste av kontorer som heter "NAV Kontaktsenter <XXX>", men usikker på om de burde vært her eller ikke?

Reply:
Dette er de jeg fikk oversendt av fageksperten for KS Anne Nordkil så det er kun disse som skal være her som KS.

Reply:
Nå er det vel ikke slik at enhetene forandrer seg brått slik som kommuner og fylker, eller hva mener du her?

Reply:
Uansett per nå har vi kun frontendvalidering som går på det samme.

Reply:
Regner med at det ikke er stadig endringer på dette nei.

Hvis du har fått en liste, så stemmer det sikkert - ble bare overrasket over at jeg fikk opp flere tilsynelatende kontaktsentere (som jeg antok kanskje skulle ha vært med).